lightpath/path.py

```diff
@@ -114,14 +114,23 @@ def find_device_state(device: Device) -> Tuple[DeviceState, LightpathState]:
     """
     # Gather device information
     try:
+        for sig in device.lightpath_summary._signals:
+            if not sig.connected:
+                # Check if relevant signals are connected
+                logger.debug(f"Unable to connect to device: {device}, "
```

Comment:
```suggestion
                logger.debug(f"Unable to connect to device: {device.name}, "
```
Device reprs can be annoyingly long